src/test/java/gov/usgs/cida/pubs/dao/PublicationDaoIT.java

```diff
@@ -355,7 +355,8 @@ public void getObjectCountTest() {
 		filters.put(PublicationDao.CONTRIBUTING_OFFICE, new String[]{"contributingOffice1", "contributingOffice2"});
 		filters.put(PublicationDao.CONTRIBUTOR, "contributor1" + MvcService.TEXT_SEARCH_STARTS_WITH_SUFFIX + MvcService.TEXT_SEARCH_AND + "contributor2" + MvcService.TEXT_SEARCH_STARTS_WITH_SUFFIX);
 		
-		filters.put(PublicationDao.DOI, true);
+		filters.put(PublicationDao.DOI, new String[]{"DOI-123"});
```

Comment:
Can you add a second doi here to show that the foreach is set up correctly? (The separator is valid.)

It also looks like the getByMapTest() in this class hasn't been truly kept up to date. There should be a separate test showing that specifying just DOI values returns the correct publications and one that shows specifying just the hasDoi returns the correct publications.  (Might also be time to split it into individual test.)